### DISCUSSION TITLE
Backup namespace without PV snapshot and only with storage account key in Azure

### DISCUSSION THREAD

**Discussion by veerendra2** [vmware-tanzu/velero#4324] [Community support Q&A]
I'm trying to setup backup/restore for Strimzi Kafka. We are using older version of K8s which dosnt support volume snapshot. Is possible backup namespace(including PV) to storage account(Azure)?
```
$ oc version
oc v3.11.0+0cbc58b
kubernetes v1.11.0+d4cacc0

Server https://openshift.dev.ocp.sirius.supplyon.com:443
openshift v3.11.0+cbab8ee-94
kubernetes v1.11.0+d4cacc0
```
## Velero Deployment
```
velero install \
	--provider azure \
	--plugins velero/velero-plugin-for-microsoft-azure:v1.1.0 \
	--bucket $BLOB_CONTAINER \
	--secret-file ./credentials-velero \
	--backup-location-config \
	resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,\
	storageAccount=$AZURE_STORAGE_ACCOUNT_ID,\
	storageAccountKeyEnvVar=AZURE_STORAGE_ACCOUNT_ACCESS_KEY,\
	subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID \
	--use-volume-snapshots=false
```
## Run Backup
```
velero backup create backup-test --include-namespaces kafka
```
I can see the backups in storage account. I believe those are only resources and there states
![image](https://user-images.githubusercontent.com/8393701/140749667-f9a4930c-adf0-4ddf-9c7b-f9f971f04757.png)

But, is there is any option to backup PV to storage account?

**Comment by zubron** [comment]
Hi @veerendra2 - When volume snapshotting capabilities are not available, we recommend using the [restic integration](https://velero.io/docs/v1.7/restic/) provided by Velero.
